locations/spiders/starbucks_jp.py

```diff
@@ -4,12 +4,11 @@
 from locations.categories import Extras, apply_yes_no
 from locations.hours import DAYS_3_LETTERS, OpeningHours
 from locations.json_blob_spider import JSONBlobSpider
-from locations.spiders.starbucks_us import STARBUCKS_SHARED_ATTRIBUTES
 
 
 class StarbucksJPSpider(JSONBlobSpider):
     name = "starbucks_jp"
-    item_attributes = STARBUCKS_SHARED_ATTRIBUTES
+    item_attributes = {"brand": "スターバックス", "brand_wikidata": "Q37158"}
```

Comment:
Personal preference not to hardcode the brand string when we can't read it. Fetch it from NSI.
```suggestion
    item_attributes = {"brand_wikidata": "Q37158"}
```